permissions/plugin-blueocean-rest.yml

```diff
@@ -17,7 +17,7 @@ developers:
 - "kshultz"
 - "bitwiseman"
 - "olamy"
-- "halkeye"
```

Comment:
Looks like your find and replace left the line in? I assume it won’t cause an issue but you could remove it

Reply:
rawr, sed is supposed to delete it. yea i'll fix

Reply:
🤦 muscle memory did /g not /d
fixed